src/GModelFit.jl

```diff
@@ -259,7 +259,7 @@ A structure containing a model description, whose evaluation is suitable to be c
 
 Constructor is: `Model(domain::AbstractDomain, components...)`
 where the first argument is either a `Domain` or `CartesianDomain` object, and the remaining one(s) is (are) the model component(s), which may be given as:
-- a single `Dict{Symbol, AbstractComponent}`, where the keys are the names and the values the component objects;
+- a single `OrderedDict{Symbol, AbstractComponent}`, where the keys are the names and the values the component objects;
```

Comment:
The `OrderedDict` is actually needed only if there are depedencies among the components being added. Hence I would suggest to leave `Dict`, and add a sentence like the following "If the components being added share some dependency, or if their order is relevant, we suggest to use `DataStructures.OrderedDict` in place of `Dict`.